docs/backplane/guideline.md

```diff
@@ -0,0 +1,94 @@
+* Existing ClusterRoles are preferred when granting permissions in a namespace.
+  * `view` - general read-only access to core Kubernetes and OpenShift resources, except for Secrets
+  * `admin` - full administrative access to the namespace.
+  * `dedicated-readers` - created by OSD/ROSA and dynamically aggregates view permissions for resources installed by OLM.
+
+Resource specific guidelines below are enforced by CI where possible.
+
+| Related Resources  | Description | Exceptions | Enforced by CI |
+| ------------------ | ----------- | ---------- | -------------- |
+| Roles/ClusterRoles | ClusterRole names must be suffixed with `-project` for namespace scoped permissions and `-cluster` for cluster scoped permissions. | `.*-aggregate`, `backplane-impersonate-cluster-admin` | :heavy_check_mark: |
+| Roles/ClusterRoles | API Groups, resources, verbs, and namespace/resource names must be explicitly defined without using glob patterns (where applicable). | apiGroups: `tekton.dev`, `logging.openshift.io`, `velero.io` | :heavy_check_mark: |
+| Roles/ClusterRoles | View access is generally permitted in `redhat-` and `openshift-` prefixed namespaces. | `openshift-backplane-cluster-admin` | :x: |
+| Roles/ClusterRoles | ClusterRoles must not include permission to get/list/watch Secrets in any namespace. | N/A | :x: |
+| Roles/ClusterRoles | All other access (non-read) must be justified by referencing a documented need (bug or missing capability). | N/A | :x: |
+| Roles/ClusterRoles | Access that would allow bypassing RBAC cannot be granted e.g. creating/execing into pods which have access to the default service account for a namespace. | N/A | :x: |
+| SubjectPermissions | SubjectPermissions must only permit ClusterRoles with `-cluster` suffix in clusterPermissions (cluster scope). | Roles: `admin`, `dedicated-readers`, `view`, `system:openshift:cloud-credential-operator:cluster-reader` | :heavy_check_mark: |
+| SubjectPermissions | SubjectPermissions must only permit ClusterRoles with `-project` suffix in permissions (namespace scope). | Roles: `admin`, `dedicated-readers`, `view`, `system:openshift:cloud-credential-operator:cluster-reader` | :heavy_check_mark: |
+| SubjectPermissions | SubjectPermissions allowed namespace regex must constrain bindings to only the subset of namespaces access is required in. | N/A | :x: |
+| SubjectPermissions | SubjectPermissions must deny the `openshift-backplane-cluster-admin` namespace. | N/A | :heavy_check_mark: |
+| config.yaml        | RBAC for Layered Products teams is only provisioned on clusters where the respective Layered Products are installed. | N/A | :x: |
+
+## Allowed Secret Access
+
+The following secrets are allowed to be accessed for the given reasons.  Note to be in this list the `Secret` must not contain any secret data (see [Definitions](#definitions)).
+
+| Namespace | Secret | Reason |
```

Comment:
nit: perhaps adding a "Who" or "Persona" column would be useful to set expectations and boundaries.

Reply:
Done, PTAL.